src/dbwrapper.h

```diff
@@ -44,6 +45,11 @@ const std::vector<unsigned char>& GetObfuscateKey(const CDBWrapper &w);
 
 };
 
+class CBitcoinLevelDBLogger : public leveldb::Logger {
```

Comment:
.h is for the externally accessible parts of a cpp file. This class doesn't need to be exposed to anything, so it can just go inside the cpp file.